addons/bus/models/bus.py

```diff
@@ -220,4 +223,6 @@ def start(self):
     # We only use the event dispatcher in threaded and gevent mode
     dispatch = ImDispatch()
     if servermod.server:
+        stop_event = threading.Event()
```

Comment:
to avoid `NameError` I would suggest we move that `stop_event` variable definition next to `dispatch = None`